crates/zoon/src/element/button.rs

```diff
@@ -8,6 +8,60 @@ use std::marker::PhantomData;
 
 make_flags!(Label, OnPress);
 
+/// Html Button for your web page.
+/// You can create a new button by using its constructor and chain styling.
+///
+/// # Examples
+/// ```no_run
```

Comment:
Do you think it would be possible to remove `no_run`? Keeping the docs and code in sync is super time-consuming and error-prone.

Reply:
I solved the issues for the examples. I tried to find a quicker way for not running the test but still compiling them and I did not find any. Not sure where to add this so 
```should we also add rust - rust, no_run ``` For now I was trying to find a way in `Cargo.toml` to compile but not run. There is a command `--no--run` but it does not make the tests even compiling. I will find  a way in next PR as you suggest :)